docs/installation.md

```diff
@@ -218,19 +218,234 @@ On Linux, as an optional post-installation task, you may wish to setup the bot t
 
 ------
 
-### Anaconda
+## Conda (Miniconda or Anaconda)
 
-Freqtrade can also be installed using Anaconda (or Miniconda).
+Freqtrade can also be installed with Miniconda or Anaconda.
+Conda is a package manager and virtual environment manager in one.
+The purpose of installing Conda (Miniconda or Anaconda) is to automatically prepare and manage the extensive library dependencies of the Freqtrade program.
 
-!!! Note
-    This requires the [ta-lib](#1-install-ta-lib) C-library to be installed first. See below.
+You can start with any clean distribution of Debian-based Linux distributions, which includes Ubuntu Linux.
+For other popular Linux distributions, see: https://distrowatch.com/ or https://www.ubuntupit.com/best-debian-based-linux-distributions/
```

Comment:
I'd keep this whole documentation platform-independent.
The strength of Conda is to work the same way on all platforms - so the tutorial should also highlight this.

Reply:
Script is changed a bit. 
I would rather not assume that everybody knows conda,